deepchem/models/torch_models/tests/test_highway.py

```diff
@@ -0,0 +1,74 @@
+import pytest
+
+try:
+    import torch
+    import numpy as np
+    has_torch = True
+except ModuleNotFoundError:
+    has_torch = False
+    pass
+
+
+@pytest.mark.torch
+def test_highway_shape_no_params():
+    from deepchem.models.torch_models.layers import Highway
+    width = 5
+    batch_size = 2
+    torch.manual_seed(42)
+
+    input = torch.rand(batch_size, width, dtype=torch.float32)
+    layer = Highway(layer_shape=[width, 2])
+    result = layer(input)
+    assert result.shape == (batch_size, width)
+    assert len(list(layer.parameters())) == 0
+
+    result2 = layer(input)
+    assert torch.allclose(result, result2)
+
+
+@pytest.mark.torch
+def test_highway_layer_non_deterministic_output():
+    from deepchem.models.torch_models.layers import Highway
+    width = 5
+    batch_size = 2
+    torch.manual_seed(42)
+
+    input = torch.rand(batch_size, width, dtype=torch.float32)
+    layer = Highway(layer_shape=[width, 2])
+    result = layer(input)
+
+    layer2 = Highway(layer_shape=[width, 2])
+    result2 = layer2(input)
+
+    assert not torch.allclose(result, result2)
+
+
+@pytest.mark.torch
+def test_highway_layer_shape():
```

Comment:
`deepchem/models/torch_models/tests/test_highway.py:73: AssertionError` Error in tests